### PR TITLE
Add new category Resources & subcategory Blogs

### LINKED ISSUE #167: [NEW LINKS] Add new category & subcategory
### What would you like to share or ask?

Add new `category ` named `Resources` and a `subcategory` named  `Blogs` and add the following Links in there:
1. https://medium.com
2. https://geeksforgeeks.com
3. https://javatpoint.com
4. https://w3schools.com
5. https://logrocket.com
6. https://tutorialspoint.com
7. https://dev.to
8. https://freecodecamp.com

### Additional information

You can write a brief description if you familiar with the site, else just take it from homepage of the site (but make sure to keep it short and concise).

### PR DESCRIPTION
**Pull request by imbeshat**
## Changes proposed

Add new category  named Resources and a subcategory named Blogs.

## Screenshots

image
image

### CODE REVIEW

File: database/data.ts [hunk 1]
```diff
@@ -479,6 +479,78 @@ export const data: IData[] = [
...
+    name: "Medium",
+    description: "Medium is a social publishing platform that is open to all where people share their writings on any topic.",
+    url: "https://medium.com",
+    category: "resources",
+    subcategory: "blogs"
+  },
+  {
+    id: "58",
+    name: "GeeksforGeeks",
+    description: "A Computer Science portal that contains well-written, well-thought, and well-explained articles.",
```
Comment (rupali-codes) [inline database/data.ts:493]:
```suggestion
    description: "GeeksforGeeks is a computer science portal for geeks offering coding and algorithm tutorials, interview questions and more.",
```

File: database/data.ts [hunk 2]
```diff
@@ -479,6 +479,78 @@ export const data: IData[] = [
...
+    name: "W3Schools",
+    description: "W3Schools is a platform that offers tutorials, references, and courses covering all aspects of web development.",
+    url: "https://www.w3schools.com",
+    category: "resources",
+    subcategory: "blogs"
+  },
+  {
+    id: "61",
+    name: "LogRocket",
+    description: "The blog section of LogRocket contains well-written blogs related to the technologies of computer science.",
```
Comment (rupali-codes) [inline database/data.ts:517]:
```suggestion
    description: "It is a frontend monitoring tool that helps with debugging by providing insights into user behavior.",
```

File: database/data.ts [hunk 3]
```diff
@@ -479,6 +479,78 @@ export const data: IData[] = [
...
+    url: "https://logrocket.com",
+    category: "resources",
+    subcategory: "blogs"
+  },
+  {
+    id: "62",
+    name: "Tutorialspoint",
+    description: "Tutorials Point is an Ed Tech platform that offers the best learning material on technical and non-technical subjects.",
```
Comment (rupali-codes) [inline database/data.ts:525]:
```suggestion
    description: "Tutorials Point is an Ed Tech platform that offers the best learning material for technical and non-technical subjects.",
```

**Review by rupali-codes — COMMENTED** [review]
Rest looks good to me. Thank you

**Comment by imbeshat** [comment]
Modified the suggested descriptions and committed the changes.
Thanks.

**Review by rupali-codes — APPROVED** [review]
TYSM

**Pull request merged**